app/views/forbiddenError.scala.html

```diff
@@ -0,0 +1,16 @@
+@*
+* This is a placeholder to allow us to write unit tests which return an error page.
+* It will be replaced once we have a properly designed error page
+*@
```

Comment:
Is this comment out-of-date? This looks like it is a properly designed error page. 🙂 

Same goes for the other pages below.